api/s2n.h

```diff
@@ -3679,6 +3679,89 @@ S2N_API int s2n_offered_early_data_accept(struct s2n_offered_early_data *early_d
 S2N_API int s2n_config_get_supported_groups(struct s2n_config *config, uint16_t *groups, uint16_t groups_count_max,
         uint16_t *groups_count);
 
+/* Indicates which serialized connection version will be provided. The default value is
+ * S2N_SERIALIZED_CONN_NONE, which indicates the feature is off. */
```

Comment:
Nit: I think we usually put documentation about specific values on those values, rather than on the whole enum? Might be worth checking how this documentation is rendered.

Reply:
We don't, it's above the enum definition.